components/testimonials/TestimonialRow.tsx

```diff
@@ -33,7 +34,12 @@ export const TestimonialRow = ({
         What do people <span className="text-sswRed">say</span>?
       </h2>
       <p className="mb-8 text-center">{tagline}</p>
-      <div className="grid justify-center gap-6 md:grid-cols-autoFit3">
+      <div
+        className={cs(
+          "grid justify-center gap-6",
+          `md:grid-cols-${testimonialResult?.length ?? 3}`
```

Comment:
Can you test it if it works with different value?

Reply:
Done